src/github.com/getlantern/yamlconf/yamlconf.go

```diff
@@ -204,6 +201,13 @@ func (m *Manager) Start() (Config, error) {
 	return m.getCfg(), nil
 }
 
+// StartPolling starts polling if there is a custom polling function defined.
+func (m *Manager) StartPolling() {
```

Comment:
I don't see why we should allow this to be called more than once.  It should either error or silently ignore further calls.